jac-uk/admin
# #1714 ACRO Template on Agency Reports

## What's included?
Export ACRO report. Here is the [ACRO form](https://app.zenhub.com/files/201485031/c5600bcc-2f59-4efc-ae51-5bd724429bba/download).

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
1. Go to Agency Reports of an exercise and click "Export data".
2. Check if data is populated.

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work

## Additional context
Demo:

https://user-images.githubusercontent.com/79906532/195061996-66231234-1cf2-43c4-8343-529c8175198e.mov

## Related permissions
- No permission changes required

---
PREVIEW:DEVELOP
